packages/ember-toucan-core/src/-private/components/lock-icon.ts

```diff
@@ -0,0 +1,11 @@
+import templateOnlyComponent from '@ember/component/template-only';
+
+export interface ToucanLockIconComponentSignature {
+  Element: SVGElement;
+  Args: {};
+  Blocks: {
```

Comment:
I could be missing something somewhere, but I tried `Blocks: never` and `glint` still passed

Reply:
Womp

Reply:
Ah! you were right! TIL! Thanks a bunch, @nicolechung !

Reply:
I'm getting this with  `Blocks: { default: never }`:

```
Type 'never' must have a '[Symbol.iterator]()' method that returns an iterator.
```

The error seems to be coming from our use of `never` rather than the fact that the consumer is passing a default block. I think the error we'd want to see would read something like "Argument of type [...] is not assignable to parameter of type 'never'".

What's less bad? Not producing a typechecking error when we should be or producing a wrong and possibly confusing error?

Reply:
Talked over Zoom - we reverted!

Reply:
<img width="801" alt="Screenshot 2023-06-26 at 2 53 41 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/771170/9f7ab048-b594-4207-af38-c7411ac7a49c">

According to the RFC it says if you have no blocks then it might work, but I'm not sure if this has been implemented.

https://rfcs.emberjs.com/id/0748-glimmer-component-signature/#blocks